ReactiveCocoa/Swift/Event.swift

```diff
@@ -63,6 +63,43 @@ public enum Event<T, E: ErrorType> {
 		}
 	}
 
+	/// Lifts the given function over the event's error.
+	public func mapError<F>(f: E -> F) -> Event<T, F> {
+		switch self {
+		case let .Next(value):
+			return .Next(value)
+
+		case let .Error(error):
+			return .Error(error.map(f))
+
+		case .Completed:
+			return .Completed
+
+		case .Interrupted:
+			return .Interrupted
+		}
+	}
+
+	/// Unwraps the contained `Next` value.
+	public var nextValue: T? {
+		switch self {
+		case let .Next(next):
+			return next.value
+		default:
+			return nil
+		}
+	}
+
+	/// Unwraps the contained `Error` value.
+	public var errorValue: E? {
```

Comment:
If we're going with `value` for next I'm fine going with `error` here. I also considered calling this `failure`.